### DISCUSSION TITLE
【web】libpag.wasm体积较大，请问如何瘦身

### DISCUSSION THREAD

**Discussion by tiny-lin** [Tencent/libpag#1949] [Q&A] [answered]
我们在实验中发现libpag.wasm体积有将近3MB，请问官方瘦身后的版本预计可以有多小，预计多久发布瘦身版本

**Comment by kevingpqi123** [comment, marked as answer]
微信小程序有两个版本：完整版 和 lite 版本https://pag.art/docs/sdk.html#%E5%BE%AE%E4%BF%A1%E5%B0%8F%E7%A8%8B%E5%BA%8F%E6%8E%A5%E5%85%A5

如果不涉及占位图替换和文本编辑，可以使用 lite 版本，需要文本编辑和占位图替换时使用完整版，你这边的 3MB 应该是没有压缩过的，可以进行下压缩
就完整版而言，目前没有办法再进一步压缩，这是由内部的实现复杂度决定的，本身这里的渲染逻辑已经足够复杂。
